index.js

```diff
@@ -881,9 +881,47 @@
   //. Just("d")
   //. ```
   var at = S.at = R.curry(function(n, xs) {
+    var endIndex = n === -1 ? -0 : n + 1;
+    return R.compose(R.map(R.head), S.slice(n, endIndex))(xs);
```

Comment:
When we use a Sanctuary function internally it's nice to use a local alias (`slice` in this case). Also, I don't see the value of `R.compose` here: since we're applying the composition immediately, we might as well nest function calls.

```javascript
return R.map(R.head, slice(n, n === -1 ? -0 : n + 1, xs));
```